FlowCrypt/Controllers/Setup/SetupBackupsViewController.swift

```diff
@@ -144,17 +144,22 @@ extension SetupBackupsViewController {
             return
         }
 
-        // save pass phrase
-        matchingKeyBackups
-            .map {
-                PassPhrase(value: passPhrase, fingerprints: $0.fingerprints)
-            }
-            .forEach {
-                passPhraseService.savePassPhrase(with: $0, inStorage: shouldStorePassPhrase)
-            }
+        if !shouldStorePassPhrase {
```

Comment:
instead of these `shouldStorePassPhrase` and `inStorage` and similar booleans, we should use an enum like `StorageMethod.permanent` vs `StorageMethod.memory`, everywhere. That would be for another PR. Then here it will say `if storageMethod == StorageMethod.memory`

Reply:
Agree, I like `enum` solution 👍 
I thought on possible improvements here, but decided that this pull request already has many changes and it'll be better to work on `inStorage` update later.